cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -192,6 +193,8 @@ def _add_login_nodes_pool_launch_template(self):
                                         NODE_BOOTSTRAP_TIMEOUT,
                                     )
                                 ),
+                                "AutoScalingGroupName": f"{self._login_nodes_stack_id}-AutoScalingGroup",
```

Comment:
We should not redefine the ASG and the hook name, but instead retrieve them from the stack, otherwise we risk to have naming misalignments.